flake.nix

```diff
@@ -0,0 +1,45 @@
+{
+  description = "Application packaged using poetry2nix";
+
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+  inputs.poetry2nix = {
+    url = "github:nix-community/poetry2nix";
+    inputs.nixpkgs.follows = "nixpkgs";
+  };
+
+  outputs = { self, nixpkgs, flake-utils, poetry2nix }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        # see https://github.com/nix-community/poetry2nix/tree/master#api for more functions and examples.
+        inherit (poetry2nix.legacyPackages.${system}) mkPoetryApplication;
+        pkgs = nixpkgs.legacyPackages.${system};
+
+      in
+      {
+        packages = {
+          myapp = mkPoetryApplication {
```

Comment:
I suggest you remove `myapp` and put the code directly under the `default` package.

Reply:
I like having a dedicated name so that `nix flake show` shows something else than "default" (the attribute on the right can be confusing).
```
    └───x86_64-linux
        ├───alot: package 'python3.10-alot-0.10'
        └───default: package 'python3.10-alot-0.10'
```